docs/pages/router/reference/api-routes.mdx

```diff
@@ -198,6 +200,19 @@ Route handlers are executed in a sandboxed environment that is isolated from the
 
 ## Deployment
 
+When you're ready to deploy to production, run [`npx expo export`](/more/expo-cli#exporting) to create the server bundle in the `dist` folder. This server can be tested locally with `npx expo serve` (available in Expo SDK 52 and higher), visit the URL in a web browser or create a native build with the `origin` set to the local server URL.
```

Comment:
```suggestion
When you're ready to deploy to production, run [`npx expo export`](/more/expo-cli#exporting) to create the server bundle in the **dist** directory. This server can be tested locally with `npx expo serve` (available in Expo SDK 52 and higher), visit the URL in a web browser or create a native build with the `origin` set to the local server URL.
```

- Let's make the dist directory name bold and use directory over folder as per our [writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)